publication-commons/src/main/java/no/unit/nva/publication/model/business/PublishingRequestCase.java

```diff
@@ -34,16 +29,15 @@
 @SuppressWarnings("PMD.GodClass")
 public class PublishingRequestCase extends TicketEntry {
     
-    public static final String RESOURCE_WITHOUT_MAIN_TITLE_ERROR = "Resource is missing main title: ";
+    public static final String RESOURCE_LACKS_REQUIRED_DATA = "Resource does not have required data to be "
```

Comment:
Can you use this in tests instead of writing the same string var everywhere?

Reply:
This was deliberate, I find it a bit better to state the concern before I update the class — it decouples the class from the test, also.